src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.js

```diff
@@ -312,14 +361,8 @@ function ComposerWithSuggestions({
      * @param {Boolean} shouldAddTrailSpace
```

Comment:
Did we get rid of this?

Reply:
@flodnv Didn't quite get the question. Do you mean did we get rid of `@param` from jsdoc? I think it has become redundant with Typescript migration.

Reply:
Sorry, I am asking: it seems that we got rid of the `shouldAddTrailSpace` variable in code, and if I'm correct it seems like we should remove this `@param` line?

Reply:
Removed the `@param` line